spec.html

```diff
@@ -26507,7 +26507,7 @@ <h1>Overview of Date Objects and Definitions of Abstract Operations</h1>
       <emu-clause id="sec-time-values-and-time-range">
         <h1>Time Values and Time Range</h1>
         <p>Time measurement in ECMAScript is analogous to time measurement in POSIX, in particular sharing definition in terms of the proleptic Gregorian calendar, an epoch of midnight at the beginning of 01 January, 1970 UTC, and an accounting of every day as comprising exactly 86,400 seconds (each of which is 1000 milliseconds long).</p>
-        <p>An ECMAScript <dfn>time value</dfn> is a Number representing an instant in time to millisecond precision as a linear sum of "day number" (the signed count of whole UTC days from the epoch to the start of the last UTC day beginning on or before the instant) weighted at 86,400,000 plus "time within the day" (the nonnegative count of milliseconds from the start of that UTC day to the instant) unweighted, or *NaN* representing no specific instant. Time values do not account for UTC leap seconds, but this definition yields piecewise alignment with UTC, discontinuities only at leap second boundaries, and zero difference outside of leap seconds.</p>
+		<p>An ECMAScript <dfn>time value</dfn> is a Number representing an instant in time to millisecond precision, or *NaN* representing no specific instant. A time value that is a multiple of <emu-eqn>24 &times; 60 &times; 60 &times; 1000 = 86,400,000</emu-eqn> (i.e., is equal to 86,400,000 &times; _d_ for some integer _d_) represents the instant at the start of the UTC day that follows the epoch by _d_ whole UTC days (preceding the epoch for negative _d_). Every other finite time value _t_ is defined relative to the greatest preceding time value _s_ that is such a multiple, and represents the instant that occurs _t_ &minus; _s_ milliseconds after the instant represented by _s_. Time values do not account for UTC leap seconds, but this definition yields piecewise alignment with UTC, discontinuities only at leap second boundaries, and zero difference outside of leap seconds.</p>
```

Comment:
Done.

Reply:
Would it be useful to be more explicit about leap seconds? Something like:

 - For any instant within a positive leap second, there is no time value that represents that  instant.
- For any instant within the second immediately following a negative leap second, there are 2 time values that represent that instant.

(If so, it might be worthwhile to pull all the leap second stuff to a separate para.)

Reply:
Done.